apps/els_lsp/src/els_typer.erl

```diff
@@ -61,7 +61,7 @@
 -record(info, {records = maps:new() :: erl_types:type_table(),
                functions = []       :: [func_info()],
                types = map__new()   :: map_dict()}).
--opaque info() :: #info{}.
+-type info() :: #info{}.
```

Comment:
does this need to be non-opaque?

Reply:
this type is being used in the spec definition of `els_code_lens_suggest_spec:get_type_spec/2`. And it seems that due to it being opaque dialyzer thinks that there might be a type conflict for this function. Not sure, though, why it worked previously.